### ISSUE TITLE
Cannot set label attribute `uses`, which makes it invalid all the time

### ISSUE THREAD

**Issue by EmCousin** [jeremytregunna/ruby-trello#249]
Hi,

First of all, thank you for writing this awesome gem.

I'd like to point out an issue that I'm having right now when I want to [add](https://github.com/jeremytregunna/ruby-trello/blob/master/lib/trello/card.rb#L402) or [remove](https://github.com/jeremytregunna/ruby-trello/blob/master/lib/trello/card.rb#L411) a label to a card.

# My problem
`label.valid?` returns `false`, because the attribute `uses` is blank. My problem is, `uses` attribute is read-only, and trying to set it at the card creation does not work either:

```ruby
label = Trello::Label.create(board_id: board_id, uses: 'uses', color: 'red', name: 'TODO')
label.uses
# => nil
```

I've tried to pass the `uses` key as a string, it won't work either
```ruby
label = Trello::Label.create(board_id: board_id, 'uses' => 'uses', color: 'red', name: 'TODO')
label.uses
# => nil
```

Any suggestions?

PS : I was able to bypass the presence validation on the attributes `:uses`, but that's not ideal since it's a workaround

**Comment by anatoliy-larin** [comment]
Hi,

I have this problem too.

Code:

```
label = Trello::Label.create(name: 'test', board_id: board.id, color: 'red')
Rails.logger.info("[Trello] Label Created: #{label} & #{label.id}")

unless label.valid?
  Rails.logger.error("[Trello] INVALID LABEL: #{label.errors.messages}")
end
```

Output

```
[Trello] Label Created: #<Trello::Label:0x00007f932f218cd8> & 5b22c063d5dd84ae9d719d72?
[Trello] INVALID LABEL: {:uses=>["can't be blank"]}
```

Help me, plz )

**Comment by EmCousin** [comment]
So far I've been removing the validator in question in the Trello initializer:


```ruby
# config/initializers/trello.rb

require 'trello'

Trello.configure do |config|
  config.developer_public_key = Rails.application.secrets.trello_developer_public_key
  config.member_token = Rails.application.secrets.trello_member_token
end

# Removing the presence validator for `uses`
trello_label_class_validators = Trello::Label.validators.find do |_validator|
  ActiveModel::Validations::PresenceValidator
end
trello_label_class_validators.attributes.reject! { |i| i == :uses }
```

It works for me, but I'd seriously like to have a hint from any maintainer of this gem to figure out why this validator exists in the first place.

**Comment by anatoliy-larin** [comment]
@EmCousin thank you for this "quick hack"! It's pragmatic fix )))

**Comment by JrmKrb** [comment]
Fixed by #254 that hopefully will be merged soon

**Comment by EmCousin** [comment]
Awesome, thanks 👍
